interface/main/calendar/add_edit_event.php

```diff
@@ -983,9 +983,10 @@ function setEventDate($start_date, $recurrence){
 <head>
 <?php html_header_show(); ?>
```

Comment:
This script has some heterogeneous controls since some use bootstrap, however, this is worth the new buttons.

Some more clean up needed:
remove lines 98, 995, 996, 997